operations/mimir-mixin/alerts/alertmanager.libsonnet

```diff
@@ -11,12 +11,12 @@
           'for': '30m',
           labels: {
             severity: 'critical',
-          },
+          } + $._config.alert_extra_labels_variable,
```

Comment:
i'm wondering if we can somehow do this automatically for any alert instead of having to copy it for every alert.

At the bottom of the file all alerts are used in a slice. Can we do something like 

```jsonnet
  
  groups+: $.withRunbookURL('https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#%s', $.withExtraLabelsAnnotations(alertGroups)),
```
and have in alert-tuils.libsonnet something along the lines of (copied from `withRunbookURL`)

```jsonnet

  withExtraLabelsAnnotations(groups)::
    local update_rule(rule) =
      if std.objectHas(rule, 'alert')
      then rule {
        annotations+: $._config.alert_extra_annotations_variable,
        labels+: $._config.alert_extra_labels_variable,
      }
      else rule;
    [
      group {
        rules: [
          update_rule(alert)
          for alert in group.rules
        ],
      }
      for group in groups
    ],
```

Reply:
will do :), I was split between doing something like that or doing it exactly at each alert individually.